### PR TITLE
feat(streaming): support value meta in mv executor

### PR DESCRIPTION
**Pull request by xx01cyx**
## What's changed and what's your intention?

Support value meta in mv executor.

## Checklist

- [x] I have written necessary docs and comments
- [x] I have added necessary unit tests and integration tests

### CODE REVIEW

File: src/stream/src/executor_v2/mview/state.rs [hunk 1]
```diff
@@ -80,16 +81,22 @@ impl<S: StateStore> ManagedMViewState<S> {
...
         let mut local = batch.prefixify(&self.keyspace);
+        let hash_builder = CRC32FastBuilder {};
 
         for (arrange_keys, cells) in self.cache.drain() {
             let row = cells.into_option();
             let arrange_key_buf = serialize_pk(&arrange_keys, &self.key_serializer)?;
             let bytes = serialize_pk_and_row(&arrange_key_buf, &row, &self.column_ids)?;
+
+            // We compute vnode on pk in materialized view since materialized views are grouped by
+            // pk.
```
Comment (fuyufjh) [inline src/stream/src/executor_v2/mview/state.rs:92]:
```suggestion
            // arrange keys.
```

After the concept 'arrange key' introduced by Chi, PK and arrange keys may be different

**Review by fuyufjh — APPROVED** [review]
LGTM

**Review by skyzh — APPROVED** [review]

**Comment by skyzh** [comment]
The mview's hash key should be `arrange_columns` w/o pk for ArrangeExecutor, and pk for MaterializeExecutor. I'll create an issue for this and fix it later.

File: src/stream/src/executor_v2/mview/state.rs [hunk 2]
```diff
@@ -80,16 +81,22 @@ impl<S: StateStore> ManagedMViewState<S> {
         let mut batch = self.keyspace.state_store().start_write_batch();
         batch.reserve(self.cache.len() * self.column_ids.len());
         let mut local = batch.prefixify(&self.keyspace);
+        let hash_builder = CRC32FastBuilder {};
```
Comment (BugenZhao) [inline src/stream/src/executor_v2/mview/state.rs:84]: We should decide the hash function globally? cc @fuyufjh

**Review by BugenZhao — APPROVED** [review]

File: src/stream/src/executor_v2/mview/state.rs (hunk 2 above)
Comment (xx01cyx) [reply src/stream/src/executor_v2/mview/state.rs:84]: If we will not introduce hash functions other than CRC32, then maybe yes.

**Pull request merged**